pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -309,7 +309,8 @@ protected void internalDeleteTopicForcefully(boolean authoritative, boolean dele
         validateNamespaceOperation(topicName.getNamespaceObject(), NamespaceOperation.DELETE_TOPIC);
 
         try {
-            pulsar().getBrokerService().deleteTopic(topicName.toString(), true, deleteSchema).get();
```

Comment:
There are several places in these "Base" classes that we call `.get()`. I think it's worth looking a bit closer to see why this is no longer working. I assume that it worked at one point.

Reply:
good idea @michaeljmarshall 

@leizhiyuan if you want to fix this specific point the best way it not to add a timeout but not perform blocking calls as described here
https://github.com/apache/pulsar/issues/14438#issuecomment-1050601917